agenta-web/src/components/pages/observability/drawer/TraceContent.tsx

```diff
@@ -356,6 +362,13 @@ const TraceContent = ({activeTrace}: TraceContentProps) => {
                     />
                 </div>
             </div>
+            {isTestsetDrawerOpen && (
+                <TestsetDrawer
+                    open={isTestsetDrawerOpen}
+                    data={[{data: activeTrace.data as KeyValuePair, key: activeTrace.key, id: 1}]}
+                    onClose={() => setIsTestsetDrawerOpen(false)}
+                />
+            )}
             {/* <Divider type="vertical" className="h-full m-0" />
```

Comment:
looks like some cleanup is needed. or is there any reason to keep this commented part?